vmm_tests/vmm_tests/tests/tests/x86_64/openhcl_uefi.rs

```diff
@@ -95,27 +95,6 @@ async fn nvme_relay_private_pool(
     nvme_relay_test_core(config, "OPENHCL_ENABLE_VTL2_GPA_POOL=512").await
 }
 
-/// Servicing test of an OpenHCL uefi VM with a NVME disk assigned to VTL2 that boots
-/// linux, with vmbus relay. This should expose a disk to VTL0 via vmbus.
-/// Use the private pool override to test the private pool dma path.
-/// Pass 'keepalive' servicing flag which impacts NVMe save/restore.
-#[openvmm_test(openhcl_uefi_x64[nvme](vhd(ubuntu_2204_server_x64))[LATEST_STANDARD_X64])]
```

Comment:
Hm, I don't think this is quite right.

This variant (openvmm with openhcl for a UEFI VM) is not currently covered by https://github.com/microsoft/openvmm/blob/0ed9ebc1d05b342aa708ef9bbbb8f8f6887cce94/vmm_tests/vmm_tests/tests/tests/multiarch/openhcl_servicing.rs#L149-L164 .

Please update the test there (`keepalive`) to include this variant.

I think we will want to take a separate follow-up to clean these up (e.g. we probably want to move all the servicing tests here to `multiarch` when possible.)